crates/volta-core/src/run/binary.rs

```diff
@@ -101,3 +107,55 @@ where
         },
     )
 }
+
+pub struct DefaultBinary {
+    pub bin_path: PathBuf,
+    pub platform: PlatformSpec,
+    pub loader: Option<BinLoader>,
+}
+
+impl DefaultBinary {
+    pub fn from_config(bin_config: BinConfig, session: &mut Session) -> Fallible<Self> {
+        let bin_path = bin_full_path(
+            &bin_config.package,
+            &bin_config.version,
+            &bin_config.name,
+            &bin_config.path,
+        )?;
+
+        // If the user does not have yarn set in the platform for this binary, use the default
+        // This is necessary because some tools (e.g. ember-cli with the --yarn option) invoke `yarn`
```

Comment:
Nit: consistency in code formatting—the same way you're wrapping `yarn`, my very detail-obsessed self wants `--yarn` to be wrapped. (This is *not* important, but it distracted me so I figured I'd note it. 😆)

Reply:
Sure, easy change!